web/Makefile

```diff
@@ -15,7 +15,7 @@ build:
 	--volume ${CURDIR}/velocity:/app \
 	--workdir /app \
 	node:8 \
-	sh -c "NODE_ENV=prod node_modules/.bin/webpack"
+	sh -c "NODE_ENV=prod ARCHITECT_ADDRESS=__ARCHITECT_ADDRESS__ node_modules/.bin/webpack"
```

Comment:
@VJftw I put this here for now, but I guess it could be moved to wherever you want…?